docs/components/compact.md

```diff
@@ -6,7 +6,7 @@ menu: components
 
 # Compactor
 
-The compactor component of Thanos applies the compaction procedure of the Prometheus 2.0 storage engine to block data stored in object storage.
+The `thanos compact` component of Thanos applies the compaction procedure of the Prometheus 2.0 storage engine to block data stored in object storage.
```

Comment:
```suggestion
The `thanos compact` command of Thanos applies the compaction procedure of the Prometheus 2.0 storage engine to block data stored in object storage.
```

Reply:
> The `thanos compact` command of Thanos applies

Sounds redundant to me. I think it would be better to do:
>The `thanos compact` command applies

or
>The `compact` command of Thanos applies